virt/node_maintenance/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,13 @@
+[id="virt-using-skip-node"]
+= Using skip node annotation
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-using-skip-node
+
+toc::[]
+
+include::modules/virt-using-skip-node.adoc[leveloffset=+1]
+
+.Additional resources:
+
+* xref:../../virt-managing-node-labeling-obsolete-cpu-models.adoc#virt-managing-node-labeling-obsolete-cpu-models[Managing node labeling for obsolete CPU models]
```

Comment:
This xref doesn't work because the relative path is incorrect. You need to include the virt and node_maintenance directories after the `../../`